isis/src/kaguya/apps/kaguyasp2isis/kaguyasp2isis.xml

```diff
@@ -3,10 +3,13 @@
 <application name="kaguyasp2isis" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://isis.astrogeology.usgs.gov/Schemas/Application/application.xsd">
 
   <brief>
-    Imports a Kaguya SP (Spectral Profiler) file
+    Imports a Kaguya SP (Spectral Profiler) file (deprecated)
   </brief>
 
   <description>
+  <p>
+    <b>This is a depricated, kaguyasp2ascii is a newer implementation of kaguyasp2isis, please use kaguyasp2ascii instead.</b> 
```

Comment:
Is there a clean way to clean way to link to another app in line? I've seen `<def>` using for glossary terms but I can't find an equivalent  for apps.

Reply:
There is no standard way to cross-reference in the docs. Normally, we just bold application names in documentation.

Reply:
It looks like some apps link to other apps with: ```<a href="../autoseed/autoseed.html" target="_blank">``` for example.  https://isis.astrogeology.usgs.gov/Application/presentation/Tabbed/overlapstats/overlapstats.html